browser-interface/packages/shared/world/positionThings.ts

```diff
@@ -52,11 +51,14 @@ export function receivePositionReport(
   cameraRotation?: ReadOnlyVector4,
   playerHeight?: number
 ) {
-  positionEvent.position.set(position.x, position.y, position.z)
+  //The 0.8 is a pivot correction moved here from renderer to have it send everytime
+  positionEvent.position.set(position.x, position.y + 0.8, position.z)
```

Comment:
what about making a variable for the magic number (I know, it explained above, but I just missed it on the first read)
`var pivotCorrectionOffset = 0.8; // moved it here from renderer to have it send everytime`

Reply:
Sounds good. Changed!